### DISCUSSION TITLE
Auto Map with POCO, relations and all fields, without mentioning all the columns in the query

### DISCUSSION THREAD

**Discussion by seddik** [maxtoroq/DbExtensions#55] [Q&A] [answered]

**Comment by seddik** [comment]
in the given example (Categorie, Product) you have mentioned how to load relations, but you must mention all fields.
also you had CategoryId and ProductId. in my case I use Id for every poco (table) so when I load using a relation ship (LEFT_JOIN), all loaded classes had the same Id (the one of the LEFT_JOIN clause)

**Comment by maxtoroq** [comment, marked as answer]
I'm not sure I understand your question correctly, but you can use `SqlSet` to load associations using `Include()`.

**Comment by seddik** [comment]
@maxtoroq I think that's it, I didn't try (see) it, I shall test it and close the issue if all is good
